sponsor-dapp/src/components/CreateContractModal.js

```diff
@@ -100,13 +100,13 @@ class CreateContractModal extends React.Component {
     const constructorParams = {
       sponsor: account,
       defaultPenalty: web3.utils.toWei("1", "ether"),
-      supportedMove: web3.utils.toWei(params.identifiers[formInputs.identifier].supportedMove, "ether"),
+      supportedMove: web3.utils.toWei(params.identifiers[formInputs.identifier].dappConfig.supportedMove, "ether"),
```

Comment:
Do we need the entire `identifiers.json` config or could we move the `dappConfig` part up a level and ignore all the other subobjects?

Reply:
Done.